### DISCUSSION TITLE
启动start-hugegraph.sh 超时

### DISCUSSION THREAD

**Discussion by zhuiyi1** [apache/incubator-hugegraph#2475] [General]
下载好tar 包后 没有改任何配置文件
操作
1、./bin/init-store.sh
![image](https://github.com/apache/incubator-hugegraph/assets/100415085/ee7cc8d2-7284-407a-8ae2-2afb3486cfcf)

2、./bin/start-hugegraph.sh
报错
Starting HugeGraphServer in daemon mode...
Connecting to HugeGraphServer (http://127.0.0.1:8080/graphs)................The operation timed out(30s) when attempting to connect to http://127.0.0.1:8080/graphs
See /home/xiaofeng/apache-hugegraph-incubating-1.2.0/logs/hugegraph-server.log for HugeGraphServer log output.

各位大佬有遇到过这种问题吗？

**Comment by imbajin** [comment]
Two point u could check first:
1. `jps -v` to check if the server process already under running status? (if yes, try `curl localhost:8080` to test it)
2. check the `hugegraph*.log` to see if there are any Exception/Errors?

And also we have met **same issue** before, u could search the **issue history**~

@dosu-bot seems u don't show in the discussion area? Is it because you need to manually turn this configuration on? Try to call u manually~
